app/views/providers/allocations/number_of_places.html.erb

```diff
@@ -0,0 +1,45 @@
+<%= content_for :page_title, "How many places would you like to request?"%>
+
+<% content_for :before_content do %>
+  <%= govuk_back_link_to(initial_request_provider_recruitment_cycle_allocations_path) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+  <%= form_with url: initial_request_provider_recruitment_cycle_allocations_path(
+                    @provider.provider_code,
+                    @provider.recruitment_cycle_year,
+                  ),
+                skip_enforcing_utf8: true,
+                method: :get,
+                builder: GOVUKDesignSystemFormBuilder::FormBuilder do |form| %>
+
+    <span class="govuk-caption-xl"><%= training_provider.provider_name %></span>
+
+    <%= form.govuk_text_field :number_of_places,
+          label: {
+            text: "How many places would you like to request?",
+            size: "xl",
+            tag: "h1",
+          },
+          pattern: "[0-9]*",
+          inputmode: "numeric",
+          width: 10,
+          id: "number-of-places-input" do %>
```

Comment:
I don't think we need to declare the id. It also breaks the association between the label and the input (i.e try click on the page heading it should set focus on the input)